src/jit/rationalize.cpp

```diff
@@ -892,10 +863,15 @@ Location Rationalizer::RewriteSimpleTransforms(Location loc)
         tree->gtBashToNOP();
     }
 
+    DuplicateCommaProcessOneTree(comp, this, loc.block, loc.tree);
+
```

Comment:
unrelated to your changes, but this function seems dumb. Why make it static and then pass 'this'? Why take a 'comp' when 'this->comp' exists? Why not take Location instead of splitting out 'tree' and 'block'?